src/test/java/com/epam/tc/hw3/AbstractChromeTest.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.tc.hw3;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import utils.Config;
+
+public class AbstractChromeTest {
+
+    protected WebDriver driver;
+    static WebDriverWait webDriverWait;
```

Comment:
Why do you declare leftMenuItems variable in base class? You only need it in Exercise1Test class.